packages/common/sagas/entities/member.js

```diff
@@ -263,7 +263,9 @@ export function* addMemberToTeamListsBatchSaga({ payload: { teamId, users } }) {
       teamId,
     });
 
-    const membersById = convertMembersToEntity(serverMembers);
+    const membersById = convertMembersToEntity(
+      serverMembers.map(member => ({ ...member, accessGranted: true })),
```

Comment:
Please move it to the normalizer that's what it for being.

Reply:
Done